jobs/src/test/scala/dcos/metronome/jobrun/impl/JobRunExecutorActorTest.scala

```diff
@@ -395,16 +401,23 @@ class JobRunExecutorActorTest extends TestKit(ActorSystem("test"))
     val activeJobRun = JobRun(JobRunId(defaultJobSpec), defaultJobSpec, JobRunStatus.Active, clock.instant(), None, None, Map.empty)
     val runSpecId = activeJobRun.id.toRunSpecId
     val runSpec: RunSpec = activeJobRun.toRunSpec
-    val queuedTaskInfo = QueuedTaskInfo(
+    val queuedTaskInfo = QueuedInstanceInfo(
       runSpec = runSpec,
-      inProgress = true,
-      tasksLeftToLaunch = 0,
-      finalTaskCount = 1,
-      tasksLost = 0,
-      backOffUntil = Timestamp(0))
+      inProgress = false,
```

Comment:
why did `true` change to `false` here? Seems that `false` is actually correct, but I wonder why it was `true` before

Reply:
I moved back to true but I think it does not matter...